src/Stratis.Bitcoin.Features.Miner.Tests/PowBlockAssemblerTest.cs

```diff
@@ -133,7 +133,7 @@ public void ComputeBlockVersion_UsingChainTipAndConsensus_Bip9DeploymentActive_U
             {
                 var newOptions = new ConsensusOptions();
                 this.testNet.Consensus.Options = newOptions;
-                this.testNet.Consensus.BIP9Deployments[0] = new BIP9DeploymentsParameters(19,
+                this.testNet.Consensus.BIP9Deployments[0] = new BIP9DeploymentsParameters("Test", 19,
```

Comment:
19 on a new line for consistency

Reply:
Fixed

Reply:
Not quite, I meant each param on a new line as per current formatting. Not a big deal, just a suggestion